### DISCUSSION TITLE
Is it possible to set GitHub Dark Colourblind (Beta) to a specific type of colour blindness?

### DISCUSSION THREAD

**Discussion by ghost** [primer/github-vscode-theme#279] [Q&A] [answered]
Hey all, 👋

I've downloaded the GitHub VS Code Theme, (great job btw). However, I was wondering if it is possible to set a specific colour style much like in the GitHub Profile Settings? ([See GitHub's Release on this subject.](https://github.blog/changelog/2021-09-29-colorblind-themes-beta/)).

<img width="320" alt="Screenshot 2022-05-11 at 18 06 49" src="https://user-images.githubusercontent.com/84478399/167907548-01aed416-ea43-4b25-b261-8828643bbed4.png">

Specifically, I make good use of the Dark Protanopia & Deuteranopia due to my visual impairment so an option to do something similar in VS Code would be awesome! 👌

Of course if this can be tweaked already, I'd love the info as I can't find anything about it on the web.

Thanks in advance. 👍

**Comment by simurai** [comment, marked as answer]
Yes, the plan is to stay in sync with https://github.com/settings/appearance and add:

- Light/Dark Protanopia & Deuteranopia
- Light/Dark Tritanopia

We'll have to think about what to do with the existing Light/Dark Colorblind themes since we can't really rename them.

**Reply by ghost** [reply]
Thanks for the reply, 'tis much appreciated. I'm not too versed in VS Code extensions and what capabilities are available to extension creators, but wouldn't it make sense to leave the name as is and have a setting for the theme itself? 

Is it possible to create custom `"workbench.colorCustomizations": {}`? 🤔

**Reply by simurai** [reply]
I think VS Code uses the extension name as identifier. So we can't really rename themes and if we would remove the current Light/Dark Colorblind themes people would fall back to the default theme and would need to manually switch to the new color blind themes.

> Is it possible to create custom "workbench.colorCustomizations": {}?

Yeah, that should be possible regardless of theme and can be used to override themes.
